samples/openthread/libraries/dts.overlay

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
We don't need board configuration dts overlay. Anyway in case if we need the file name should be fixed.